test/explorer/series_test.exs

```diff
@@ -6578,9 +6578,14 @@ defmodule Explorer.SeriesTest do
 
     test "extracts primitive from json and nil for mismatch" do
```

Comment:
should we change the name of this test?

Reply:
done: [065776f](https://github.com/elixir-explorer/explorer/pull/1031/commits/065776ff4a7664a0aa94d67bca83eab88074635f)